apps/product/tasks.py

```
@@ -1,207 +1,983 @@
+        results1 = get_yandex_orders(api_key_bearer, date_from,client_id=fby_campaign_id)
+        results2 = get_yandex_orders(api_key_bearer, date_from,client_id=fbs_campaign_id)
+        
+        results = results1 + results2
+        
+        for item in results:
+            
+            buyer_total = item.get("buyerTotal",0)
+            item_total = item.get("itemsTotal",0)
+
+            if buyer_total == item_total:
+                if "serviceName" in item["delivery"].keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)